src/main/java/mods/eln/Eln.java

```diff
@@ -2743,7 +2785,7 @@ private void registerTurbine(int id) {
             name = TR_NAME(Type.NONE, "200V Turbine");
             double RsFactor = 0.10;
             double nominalU = MVU;
-            double nominalP = 500 * heatTurbinePowerFactor;
+            double nominalP = 2000 * heatTurbinePowerFactor; // it was 2000 before
```

Comment:
This is a compatibility branch to the main repo, while it may be active on your build, this one's intended to go into the main build, which is why it needs to be defined.